### PR TITLE
build: switch to fbw eslint config

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of changes

* Switch to `@flybywiresim/eslint-config`
* Fix new lint errors

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,63 +1,46 @@
...
+        "jsx-a11y/no-static-element-interactions": "off",
+        "jsx-a11y/click-events-have-key-events": "off",
+        "jsx-a11y/anchor-is-valid": "off",
+        "object-curly-newline": ["error", { "multiline": true }]
+    },
+    "globals": {
+        "Simplane": "readonly",
+        "SimVar": "readonly",
+        "Utils": "readonly"
     }
```
Comment (NathanInnes) [inline .eslintrc.json:45]: Are these entirely necessary for the lint plugin or can they be removed

File: src/components/legal/TermsOfService.tsx [hunk 2]
```diff
@@ -50,24 +52,42 @@ export const TermsOfService: React.FC = () => {
...
-                    Flybywire Simulations reserve the right to remove users under the age of 13 and minimum age of digital consent
-                    in any given case from communication platforms.</p>
+                    <p className="text-xl mt-4 leading-9 font-light">
+                        To protect the privacy of younger people it is very important that we implement an age restriction of
+                        <b>
+                            13 and
+                            over
+                        </b>
+                        {' '}
+                        to join Flybywire Simulation services such as this website and Discord or any other communication
```
Comment (NathanInnes) [inline src/components/legal/TermsOfService.tsx:62]: Some funny business going down here. Some text is being split with { ' ' } as a replacement for space.

File: src/components/projects/A32NX/Download.tsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ export const Download: React.FC = () => {
     const urls = {
         stable: 'https://flybywiresim-packages.b-cdn.net/stable/A32NX-stable.zip',
         dev: 'https://flybywiresim-packages.b-cdn.net/vmaster/A32NX-master.zip',
-        exp: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip'
+        exp: 'https://flybywiresim-packages.b-cdn.net/vmaster-cfbw-cap/A32NX-master-cfbw-cap.zip',
```
Comment (NathanInnes) [inline src/components/projects/A32NX/Download.tsx:8]: Adding extra commas where its not needed

Comment (Benjozork) [reply src/components/projects/A32NX/Download.tsx:8]: no we require trailing commas in the config

it's style

Comment (Benjozork) [reply src/components/projects/A32NX/Download.tsx:8]: deal with it

Comment (Benjozork) [reply src/components/projects/A32NX/Download.tsx:8]: bruh

File: src/components/legal/TermsOfService.tsx (hunk 2 above)
Comment (Benjozork) [reply src/components/legal/TermsOfService.tsx:62]: This is normal so that the tags can be on their own lines, otherwise there is no spacing

**Pull request merged**